datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -932,17 +932,17 @@ fn round_trip_scalar_values() {
         ScalarValue::Binary(None),
         ScalarValue::LargeBinary(Some(b"bar".to_vec())),
         ScalarValue::LargeBinary(None),
-        ScalarValue::from((
-            vec![
```

Comment:
FWIW this would have panic'd previously if the input was invalid. Using the `ScalarStructBuilder` makes it clear that this can happen